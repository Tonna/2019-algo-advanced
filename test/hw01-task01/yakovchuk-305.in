25
1 1 2 2 4 1 5 2 2 7 9 4 11 3 11 5 1 14 15 7 8 7 8 1
3
5 10
18 1
12 8
