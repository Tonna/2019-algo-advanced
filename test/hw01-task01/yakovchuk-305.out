24
18
35
